src/Controls/src/Core/HandlerImpl/Window.iOS.cs

```diff
@@ -1,26 +1,12 @@
 ﻿#nullable enable
-
 using System;
-using System.Collections.Generic;
-using System.Text;
-using Microsoft.Maui.Controls.Platform;
 using UIKit;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Window
 	{
-		internal UIWindow? NativeWindow
-		{
-			get
-			{
-				if (Page?.Handler?.NativeView is UIView view)
-				{
-					return view.Window;
-				}
-
-				return null;
-			}
-		}
+		internal UIWindow? NativeWindow =>
```

Comment:
We can get rid of the nullable on the return here right? Since we are just throwing if it's null?

Reply:
```suggestion
		internal UIWindow NativeWindow =>
```